docs/quickstart.md

```diff
@@ -165,6 +142,26 @@ STATUS: DEPLOYED
 
 ## 3. Use Helm to install Coherence
 
+Oracle Coherence docker image will be pulled by the Helm chart from
```

Comment:
How about "By default the Oracle Coherence Docker image pulled by the Coherence Helm chart is from the Oracle Container Registry" ?